src/main/java/util/RandomSymbolUtil.java

```diff
@@ -0,0 +1,27 @@
+package util;
+
+import com.github.javafaker.Faker;
+
+/***
+ * Random generator of strings
+ */
+public class RandomSymbolUtil {
+
+    static Faker faker = new Faker();
+
+    public static String getRandomLogin() {
+        return faker.internet().emailAddress();
+    }
+
+    public static String getRandomPassword() {
+        return faker.internet().password();
+    }
+
+    public static String getRandomQuery() {
+        return faker.internet().ipV6Address();
+    }
+
+    public static String getNoCredentials() {
```

Comment:
please make this method more general. This one can be useful only in one or two tests, no point in creating a separate method for that

Reply:
done